graph-refresh/overlays/test/cronjob.yaml

```diff
@@ -41,7 +43,7 @@ spec:
                   value: "https://kubernetes.default.svc.cluster.local"
                 - name: KUBERNETES_VERIFY_TLS
                   value: "0"
-                - name: THOTH_INFRA_NAMESPACE
+                - name: NAMESPACE
```

Comment:
I think it's good, I see it's being used at https://github.com/thoth-station/graph-refresh-job/blob/master/producer.py#L60.